en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md

```diff
@@ -0,0 +1,368 @@
+
+### Actuator Setup
+
+:::warning
+Make sure the props are removed!
+The motors are easy to start in the actuators tab by accident.
+:::
+
+Motors, control surfaces, and other actuators are configured in the QGroundControl [Actuator Configuration & Testing](../config/actuators.md).
+
+The [parameter file](#load-parameters-file) loaded previously means that this screen should already be correctly setup: you just need to adjust the trims for your particular vehicle.
+If motors/servos were connected to different outputs than suggested, you will need to change the output to function mappings in the actuator output section.
+
+#### Tilt Servos
+
+1. Switch the vehicle into manual mode (either via the flight mode switch or type `commander mode manual` into the MAVLink shell).
+1. Check if the motors point upwards.
+   If the motors point forwards then their associated Tilt servos need to be reversed (selecting the checkbox next to each servo).
+
+   ![Tilt Servo adjustment](../../assets/airframes/vtol/omp_hobby_zmo_fpv/tilt-limits-01.jpg)
+
+1. Adjust the minimum or maximum value that the servo is pointing vertical up.
+1. Then type `commander transition` into the MAVLink shell to adjust the horizontal position.
+
+#### Control Surfaces
+
+Check if the actuators need to be reversed using the RC-Controller:
+
+- Roll stick to the right. The right aileron should go up, left aileron should go down.
+- Pitch stick to the back (fly upwards). Both V-tail surfaces should move up.
+- Yaw stick to the right. Both surfaces should move to the right
+
+Now adjust the trim value so that all the surfaces are in neutral position.
+
+![Servo trim](../../assets/airframes/vtol/omp_hobby_zmo_fpv/servo_trim.png)
+
+#### Motor Direction and Orientation
+
+Make sure the props are removed!!
+
+- `Motor 1`: Front left motor should spin CW
+- `Motor 2`: Front right motor should spin CCW
+- `Motor 3`: Rear motor should spin CCW
+
+If the motor spins in the wrong directions two of the three motor wires need to be swapped.
+The direction can't be changed in software because the vehicle does not use [DShot ESC](../peripherals/dshot.md).
+
+## First Flight
+
+- Check CG (There are markings underneath the wing)
+- Check actuator orientations and neutral trim
+- Check tilt rotor reactions in [Stabilized mode](../flight_modes_fw/stabilized.md)
+  - Yaw the vehicle to the right (nose to the right). The right motor should tilt down.
+  - Yaw the vehicle to the left (nose to the left). The left motor should tilt down.
+- Check tilt rotor reactions in [Stabilized mode](../flight_modes_fw/stabilized.md)
+  - Roll the vehicle to the right. The right aileron should go down.
+  - Pitch the vehicle up (nose up). Both elevons should go down.
+  - Yaw the vehicle to the right (nose to the right). Both elevons should go to the right.
+- Check kill switch

```

Comment:
Why? How? A beginner might flight to 300m then press this. Do you think we could provide some more advice?